RecoBTag/ChargeTagging/src/CandidateChargeBTagComputer.cc

```diff
@@ -0,0 +1,228 @@
+#include "RecoBTag/ChargeTagging/interface/CandidateChargeBTagComputer.h"
+
+CandidateChargeBTagComputer::CandidateChargeBTagComputer(const edm::ParameterSet & parameters):
+  useCondDB_(parameters.existsAs<bool>("useCondDB") ? parameters.getParameter<bool>("useCondDB") : false),
```

Comment:
can we define fillDescriptions here?
existsAs can be used only in exceptional cases (e.g., some large base of configs exists which should be changed)
Since this is a new code, I suggest not to use existsAs and define all needed values in the configuration file